data/events.json

```diff
@@ -822,5 +822,133 @@
     "end": "2023-04-19 23:59",
     "groupId": 10,
     "filterId": 5
+  },
+  {
+    "title": "[CESIUM] Capture The Flag",
```

Comment:
```suggestion
    "title": "[CeSIUM] Capture The Flag",
```

Reply:
Wasn't it supposed to be all capital letters by definition? :eyes:

Reply:
![image](https://user-images.githubusercontent.com/93675410/223697186-ce6c731a-b2a4-441d-b536-75a668579963.png)